### PR TITLE
Backend Product Attribute Date Design is not proper.

### PR DESCRIPTION
**Pull request by SyntaxKay**
### Description (*)

Magento 2.3 Develop

### Manual testing scenarios (*)

1. Login to backend > Store > Attribute > Product
2. Under the properties > Attribute Properties > Catalog Input Type for Store Owner (Select Date)
3.Under Advanced Attribute Properties > Default Value Check Date input design.

### Expected Result (*)
expactedss

### Actual Result Result (*)
actual-ss

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @speedy008. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by dmytro-ch** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, here is your [new Magento instance](https://pr-21067.instances.magento-community.engineering).
Admin access: [https://pr-21067.instances.magento-community.engineering/admin](https://pr-21067.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-4195](https://jira.corp.magento.com/browse/ENGCOM-4195) has been created to process this Pull Request

**Comment by sivaschenko** [comment]
Hi @speedy008 , thanks for the pull request. Actual result looks just fine. What is the issue?

**Comment by SyntaxKay** [comment]
@sivaschenko  In backend through out all date input design should be consistence and here calendar button is out side input box.

File: app/design/adminhtml/Magento/backend/web/css/styles-old.less [hunk 1]
```diff
@@ -1076,29 +1076,6 @@
         margin-top: 4px;
     }
 
-    .control {
```
Comment (sivaschenko) [inline app/design/adminhtml/Magento/backend/web/css/styles-old.less:1079]: I have a concern regarding backward compatibility aspects of removing these styles. Extensions may relay on these styles. I'd try to find a fix without removing this less block

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @speedy008 thanks for the pull request! Please see my code review comment.

**Comment by sidolov** [comment]
@speedy008 , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**